blocks/embed/embed.js

```diff
@@ -175,10 +175,10 @@ const loadEmbed = (block) => {
     // loading embed function for given config and url.
     if (config) {
       a.outerHTML = config.embed(url);
-      block.classList = `block embed embed-${config.type}`;
+      block.classList = `block embed block-visible embed-${config.type}`;
```

Comment:
```suggestion
      block.classList.add(`embed-${config.type}`);
```
I would update this as adding class than overwriting existing classlist. block-visible class is supposed to be added automatically once it gets loaded.  This was a new stuff added from Helix backend after I finished this block.
This is something I referenced from other helix site and surely was bad code.